From 4413fa36189008c776c04a4dc2150c5e1bd1ef15 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Fri, 21 Oct 2022 12:28:26 +0200
Subject: [PATCH] remove unnecessary (and incorrect) version check

k3b already depends on kf5.88, no point having code for older versions
---
 src/CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 09a59f3..20ced14 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -269,11 +269,7 @@ install( FILES  k3b.notifyrc DESTINATION ${KDE_INSTALL_KNOTIFY5RCDIR} )
 install( FILES  k3bui.rc k3bdeviceui.rc DESTINATION ${KXMLGUI_INSTALL_DIR}/k3b )
 install( FILES  org.kde.k3b.appdata.xml DESTINATION ${KDE_INSTALL_METAINFODIR} )
 
-if (${ECM_VERSION} STRGREATER "5.58.0")
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/option/k3btheme.knsrc DESTINATION ${KDE_INSTALL_KNSRCDIR})
-else()
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/option/k3btheme.knsrc DESTINATION ${CONFIG_INSTALL_DIR})
-endif()
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/option/k3btheme.knsrc DESTINATION ${KDE_INSTALL_KNSRCDIR})
 
 set( SHARED_MIME_INFO_MINIMUM_VERSION "0.23" )
 find_package( SharedMimeInfo REQUIRED )
-- 
2.38.1

